### PR TITLE
Fix boolean settings

### PR DESCRIPTION
**Pull request by sobakasu**

### CODE REVIEW

File: lib/has_settings/has_settings/shared_methods.rb [hunk 1]
```diff
@@ -61,7 +61,7 @@ def setting_value(setting)
       when "image"
         setting.file
       when "boolean"
-        !setting.value.to_s.eql?("1").nil?
+        setting.value.to_s.eql?("1")
```
Comment (sfnelson) [inline lib/has_settings/has_settings/shared_methods.rb:64]: What about using

```
ActiveModel::Type::Boolean.new.cast(setting.value)
```

File: spec/factories/settings.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    prefix { "page" }
+
+    trait :string do
+      field_type { "string" }
+      value { Faker::Lorem.word }
+    end
+
+    trait :boolean do
+      field_type { "boolean" }
+      value { "1" }
```
Comment (sfnelson) [inline spec/factories/settings.rb:20]: yay tests

**Review by sfnelson — APPROVED** [review]

File: lib/has_settings/has_settings/shared_methods.rb (hunk 1 above)
Comment (sobakasu) [reply lib/has_settings/has_settings/shared_methods.rb:64]: good idea, will use that

File: spec/factories/settings.rb (hunk 2 above)
Comment (sobakasu) [reply spec/factories/settings.rb:20]: 🙂

**Pull request merged**